src/javascript/app/pages/user/set_currency.js

```diff
@@ -152,6 +152,10 @@ const SetCurrency = (() => {
             const $clicked_currency = $(this);
             $currency_list.find('> div').removeClass('selected');
             $clicked_currency.addClass('selected');
+            
+            if ($submit) {
+                $submit.removeClass('button-disabled');
```

Comment:
we shouldn't be doing this here, this is on line 275 right before showing the error, so we should figure out why that line isn't working and fix that instead